epochStart/metachain/systemSCs.go

```diff
@@ -311,6 +366,9 @@ func (s *systemSCProcessor) getSortedJailedNodes(validatorInfos map[uint32][]*st
 
 	sort.Slice(jailedValidators, func(i, j int) bool {
 		if jailedValidators[i].TempRating == jailedValidators[i].TempRating {
```

Comment:
There is a wrongly use of "i" instead "j" in the right side of this comparation. Probably an old bug. Please check if there exists other places where these sorts with TempRating are used with this pattern -> i, i

Reply:
Done.